README.md

```diff
@@ -17,6 +16,8 @@ npm install screenie-server
 
 Alternatively, we provide a Docker image (build from the
 [Dockerfile](Dockerfile)) at [eliksir/screenie-server](https://hub.docker.com/r/eliksir/screenie-server/).
+BUT that container is running by default in sandbox mode since the image doesn't
```

Comment:
Err, the issue is that we are opting *out* of sandbox mode. Ideally we would be running in sandbox mode if the Docker image were to support user namespaces.

Reply:
Wopsi yeah of course. Ill update.